tests/helpers.py

```diff
@@ -230,3 +239,158 @@ def get_admin_key():
     out = ct_pod.exec_ceph_cmd('ceph auth get-key client.admin')
     base64_output = base64.b64encode(out['key'].encode()).decode()
     return base64_output
+
+
+def get_cephfs_data_pool_name():
+    """
+    Fetches ceph fs datapool name from Ceph
+
+    Returns:
+        str: fs datapool name
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    out = ct_pod.exec_ceph_cmd('ceph fs ls')
+    return out[0]['data_pools'][0]
+
+
+def validate_cephfilesystem(fs_name):
+    """
+     Verify CephFileSystem exists at ceph and k8s
+     Args:
+        fs_name (str): The name of the Ceph FileSystem
+    """
+    CFS = ocp.OCP(
+        kind=constants.CEPHFILESYSTEM,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
+    ct_pod = pod.get_ceph_tools_pod()
+    ceph_validate = False
+    k8s_validate = False
+    cmd = "ceph fs ls"
+    logger.info(fs_name)
+    out = ct_pod.exec_ceph_cmd(ceph_cmd=cmd)
+    if out:
+        out = out[0]['name']
+        logger.info(out)
+        if out == fs_name:
+            logger.info("FileSystem got created from Ceph Side")
+            ceph_validate = True
+        else:
+            logger.error("FileSystem was not present at Ceph Side")
+            return False
+    result = CFS.get(resource_name=fs_name)
+    if result['metadata']['name']:
+        logger.info(f"Filesystem got created from kubernetes Side")
+        k8s_validate = True
+    else:
+        logger.error("Filesystem was not create at Kubernetes Side")
+        return False
+    return True if (ceph_validate and k8s_validate) else False
+
+
+def get_all_storageclass_name():
+    """
+    Function for getting all storageclass
+    :return list of storageclass
+    """
+    SC = ocp.OCP(
+        kind=constants.STORAGECLASS,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+
+    storageclass = [
+        item.get('metadata').get('name') for item in sample if (
+            item.get('metadata').get('name') not in constants.IGNORE_SC
+        )
+    ]
+    return storageclass
+
+
+def delete_all_storageclass():
+    """"
+    Function for Deleting all storageclass
```

Comment:
fix - add return value
Please go over all docstrings and fix all according to the coding guidelines

Reply:
Done fixed doc string